package.json

```diff
@@ -114,6 +114,7 @@
     "@salesforce/sf-plugins-core": "^5.0.10",
     "@salesforce/ts-types": "^2.0.9",
     "chalk": "^5.3.0",
+    "change-case": "^5.3.0",
```

Comment:
I see `change-case` v4 in the current lockfile, can't find the changelog in the repo
https://github.com/blakeembrey/change-case

<img width="1194" alt="Screenshot 2023-12-15 at 12 22 06" src="https://github.com/salesforcecli/plugin-data/assets/6853656/11af5d7d-509b-47c4-8d61-e9f74dc31e96">

Reply:
we didn't find any breaking change that affects us going v4 -> v5.